extensions/optaplanner-jsonb/deployment/pom.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <groupId>io.quarkus</groupId>
+        <artifactId>quarkus-optaplanner-jsonb-parent</artifactId>
+        <version>999-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>quarkus-optaplanner-jsonb-deployment</artifactId>
+    <name>Quarkus - OptaPlanner JSON-B - Deployment</name>
+
+    <dependencies>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-core-deployment</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-arc-deployment</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-jsonb-spi</artifactId>
```

Comment:
Shouldn't this be a dependency on `quarkus-jsonb-deployment` instead?

Reply:
The `quarkus-jsonb-spi` dependency is needed in the deployment module according to the [docs](https://quarkus.io/guides/writing-extensions#customizing-json-b). But I am missing the `quarkus-jsonb-deployment` dep here, fixing in next commit.

Reply:
Well, the SPI should come with the deployment module.

Reply:
Good point. Sorry, didn't check :/ Fixing in next push.

Perhaps the docs should instead say to add `quarkus-jsonb-deployment` rather than `quarkus-jsonb-spi`? Same goes for `quarkus-jackson-deployment`. I can make a PR for that.

Reply:
Well, not really. You want to use the SPI if your dependency to Jackson/JSON-B is optional (which is often the case).

Here you really want a hard dependency so you don't have to use the SPI: you need the whole extension.